vagrant/scripts/generate-local-values-file.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+cat > /sumologic/vagrant/values.local.yaml <<END
```

Comment:
We shouldn't override existing file

Reply:
Hm why not? When you build stuff you override the old build dir.

Reply:
I want to keep some configuration for test without need to change it with every build. There would be no difference with using `vagrant/values.yaml` if we keep overriding this file automatically. Anyway this is not happening right now as I see. You have to manually fire the script. Its's not automated anywhere